tutorials/misc/plot_ecog.py

```diff
@@ -9,71 +11,114 @@
-# by the fact that below we do not specicty a trans file so the Head<->MRI
-# transform is the identity.
-montage = mne.channels.make_dig_montage(ch_pos=dict(zip(ch_names, elec)),
-                                        coord_frame='head')
+montage = mne.channels.make_dig_montage(ch_pos, coord_frame='head')
 print('Created %s channel positions' % len(ch_names))
 
 ###############################################################################
-# Now that we have our electrode positions in MRI coordinates, we can create
-# our measurement info structure.
+# Now that we have our montage, we can load in our corresponding
+# time-series data and set the montage to the raw data.
+
+# first we'll load in the sample dataset
+raw = mne.io.read_raw_edf(misc_path + '/ecog/sample_ecog.edf')
+
+# drop bad channels
+raw.info['bads'].extend([ch for ch in raw.ch_names if ch not in ch_names])
+raw.load_data()
+raw.drop_channels(raw.info['bads'])
+
+# attach montage
+raw.set_montage(montage)
 
-info = mne.create_info(ch_names, 1000., 'ecog').set_montage(montage)
+###############################################################################
+# We then compute the signal power in certain frequency bands (e.g. 30-90 Hz).
+# We compute the power in gamma and alpha bands.
+gamma_power = np.sum(raw.copy().filter(30, 90).get_data() ** 2, axis=1)
+alpha_power = np.sum(raw.copy().filter(8, 12).get_data() ** 2, axis=1)
 
 ###############################################################################
 # We can then plot the locations of our electrodes on our subject's brain.
 #
 # .. note:: These are not real electrodes for this subject, so they
 #           do not align to the cortical surface perfectly.
 
-subjects_dir = mne.datasets.sample.data_path() + '/subjects'
-fig = plot_alignment(info, subject='sample', subjects_dir=subjects_dir,
+subjects_dir = sample_path + '/subjects'
+fig = plot_alignment(raw.info, subject='sample', subjects_dir=subjects_dir,
                      surfaces=['pial'])
 mne.viz.set_3d_view(fig, 200, 70)
 
 ###############################################################################
 # Sometimes it is useful to make a scatterplot for the current figure view.
 # This is best accomplished with matplotlib. We can capture an image of the
 # current mayavi view, along with the xy position of each electrode, with the
-# `snapshot_brain_montage` function.
+# `snapshot_brain_montage` function. We can then visualize for example the
+# gamma power on the brain using MNE and matplotlib functions.
 
 # We'll once again plot the surface, then take a snapshot.
-fig_scatter = plot_alignment(info, subject='sample', subjects_dir=subjects_dir,
-                             surfaces='pial')
+fig_scatter = plot_alignment(raw.info, subject='sample',
+                             subjects_dir=subjects_dir, surfaces='pial')
 mne.viz.set_3d_view(fig_scatter, 200, 70)
 xy, im = snapshot_brain_montage(fig_scatter, montage)
```

Comment:
this causes the mayavi scene to show up a second time, at the end of the tutorial. WDYT about doing the snapshotting step when the first electrode location plot is made?  And adding a sentence like `We'll also use :func:`~mne.viz.snapshot_brain_montage` to save the plot as image data, so that later in the tutorial we can plot data on top of it.`